airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/DefaultTyperDeduper.java

```diff
@@ -180,11 +223,38 @@ public CompletableFuture<Optional<Exception>> typeAndDedupeTask(StreamConfig str
               originalName);
           return Optional.empty();
         }
-        final String suffix = getFinalTableSuffix(streamConfig.id());
-        final String sql = sqlGenerator.updateTable(streamConfig, suffix);
-        destinationHandler.execute(sql);
+
+        final boolean run;
```

Comment:
lines 227-255 are identical to 179-206 (give or take some spacing) would suggest either

- creating a helper method called by both
- having `typeAndDedupeTask` just call `typeAndDedupe` (line 170)

Reply:
ugh, I thought I fixed this. this was a bad merge, I'd intended to do your second bullet

Reply:
fixed, how's this look?